subcommand/mesh-init/command.go

```diff
@@ -154,14 +158,57 @@ func (c *Command) realRun(log hclog.Logger) error {
 
 	log.Info("service and proxy registered successfully", "name", serviceName, "id", serviceID)
 
+	// Grab CA root from the Consul server if TLS is enabled.
```

Comment:
Why not disable CA verification for Envoy since it's talking localhost to Consul so there's no MITM risk.

Reply:
How do you disable TLS verification for envoy through the Consul cli? I haven't found anything in the consul codebase that would read `CONSUL_HTTP_SSL_VERIFY` env var for the `consul connect envoy -bootstrap` command or a flag that would do similar for that command.

Reply:
So for the initial call that that command makes itself to Consul, I think it respects the env var. The problem is that the generated bootstrap config doesn't have verification turned off. I edited the generated config in my branch to work around this: https://github.com/hashicorp/consul-ecs/blob/secure/subcommand/mesh-init/command.go#L209-L226